src/utils/exr-parser.worker.js

```diff
@@ -17,102 +17,104 @@
  *  };
  */
 
-let openEXRLoaded = false;
-let queuedJobs = [];
-let OpenEXR;
+exports.__esModule = true;
 
-self.addEventListener('message', (event) => {
+var EXR = require("../exr-wrap/exr-wrap.js");
```

Comment:
This is an important change as previously the EXR was being bound as a global var I believe, much better to require it as a module like this.

Reply:
I faintly remember having problems with asynchronous loading / compiling of the webasm code. These variables were used to detect whether the wasm is already loaded and available or not, and to queue up decode requests if it's not available.

Reply:
Closing this and moving convo to https://github.com/disneyresearch/jeri/pull/17/files. But in general, I think this should just work now with the require like this.